src/layers/src/Pane.js

```diff
@@ -25,7 +25,7 @@ export default class Pane extends PureComponent {
     activeElevation: ElevationPropType,
 
     // Enable to set a boolean for a default border
-    borderTop: StringAndBoolPropType,
+    border: StringAndBoolPropType,
```

Comment:
Why was `borderTop` removed?

Reply:
Shoot, i meant to add that. My bad, readding